### PR TITLE
fix: make busybox reproducible

### PR DESCRIPTION
**Pull request by thesayyn**
With https://github.com/GoogleContainerTools/distroless/pull/1538 i have discovered that busybox archives are non-reproducible.  

Its uid/gid mtime and permissions are directly affected by Bazel and the Host. 

Screenshot 2024-03-05 at 3 54 35 PM

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -2,6 +2,22 @@ workspace(name = "distroless")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+# rules_distroless setup
```
Comment (loosebazooka) [inline WORKSPACE:5]: should this workspace file be in this change?

**Review by loosebazooka — COMMENTED** [review]
lgtm except this WORKSPACE change?

File: WORKSPACE (hunk 1 above)
Comment (thesayyn) [reply WORKSPACE:5]: Yea as it being Bazel lib which has the tar rule. I can bring in Bazel lib directly but I will change it once we start using rules distroless. Do you prefer I bring in Bazel lib directly?

Comment (loosebazooka) [reply WORKSPACE:5]: Oh I see, yeah this is fine.

**Review by loosebazooka — APPROVED** [review]

**Pull request merged**